psi4/driver/qcdb/intf_dftd3/dashparam.py

```diff
@@ -0,0 +1,598 @@
+#
+# @BEGIN LICENSE
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2018 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This file is part of Psi4.
+#
+# Psi4 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+"""Module to hold and distribute the -D dispersion correction parameters."""
+
+import copy
+import collections
+
+from ..exceptions import ValidationError
+
+## ==> Dispersion Aliases and Parameters <== ##
+
+# The dashcoeff dict below defines the -D parameters for most of the DFT methods. Some 'd2' are
+#   taken from already defined functionals in psi4. Other parameters taken from the
+#   references indicated in the "citation" parameter. The remainder of the parameters are
+#   from http://toc.uni-muenster.de/DFTD3/ on September 25, 2012, with the dict keys
+#   translated from Turbomole to Psi4 functional names.
+#   Note: most DSD-functionals have their dispersion correction defined in dict_dh_funcs.py
+dashcoeff = {
+    'd2': {
+        'formal'            : "D2",
+        'alias'             : ['d'],
+        'description'       : "    Grimme's -D2 Dispersion Correction",
+        'citation'          : "    Grimme, S. (2006), J. Comp. Chem., 27: 1787-1799",
+        'bibtex'            : "Grimme:2006:1787",
+        'default'           : collections.OrderedDict([('s6', 1.0), ('alpha6', 20.0), ('sr6', 1.1)]),
+        'definitions': {
+            'blyp'          : {'s6': 1.2,  'alpha6': 20.0, 'sr6': 1.1},
+            'bp86'          : {'s6': 1.05, 'alpha6': 20.0, 'sr6': 1.1},
+            'b97'           : {'s6': 1.25, 'alpha6': 20.0, 'sr6': 1.1},  # formerly b97-d
```

Comment:
I don't understand why this was necessary; the situation with B97, B97-1 and B97-D is rather confusing as is...

Reply:
Thanks for clarifying. I suppose now that your thought process is documented on github, we'll be able to remember what's going on once we forget.